articles/100_ament.md

```diff
@@ -20,34 +20,93 @@ author: '[Dirk Thomas](https://github.com/dirk-thomas)'
 Original Author: {{ page.author }}
 
 
+## Preface
+
+ROS is developed with a federated model.
+While this comes with a lot of advantages it makes building a set of interdependent packages more complex.
```

Comment:
I think either `it make building a set of interdependent packages more complicated` or `it makes the process of building several interdependent package more complex` sounds better.